### ISSUE TITLE
Feature Request: Mega Menu

### ISSUE THREAD

**Issue by nicocollu** [soflyy/oxygen-bugs-and-features#928]
**Describe the feature you'd like to see included in Oxygen.**
I'd love to see the implementation of a Mega Menu in Oxygen. The ability to add your elements/modules inside a dropdown link such as the UberMenu plugin. Adding something such as a "Bestselling Products" with a dynamic list of products in a dropdown would be amazing.

**What are the use cases for this feature?**
eCommerce, Blogs and more.

**Examples of this feature or functionality.**
Here is a great example: https://quadmenu.com/divi/

**Comment by alriksson** [comment]
https://crocoblock.com/blog/jetmenu-2-0-more-than-mobile-friendly-version/
Demo: https://crocoblock.com/woocommerce/jetmenu/

**Comment by nicocollu** [comment]
> https://crocoblock.com/blog/jetmenu-2-0-more-than-mobile-friendly-version/
> Demo: https://crocoblock.com/woocommerce/jetmenu/

Thank you for the plugin suggestion. I have UberMenu although would love to see a themebuilder add this in-house.

**Comment by KarzanTT** [comment]
+1 Add Mega Menu feature in Oxygen Pro Menu!
https://www.megamenu.com/

**Comment by michacassola** [comment]
Yes, please add a mega menu where you can fill the box that appears with anything just like with a modal etc.

**Comment by KarzanTT** [comment]
Yes, Please Add Mega Menu feature to Oxygen,

**Comment by lbdsgn** [comment]
To tie menga menus into the WP menu system would be amazing. The current workarounds are client-unfriendly.
